File: java assignment/Day2assignment/Student.java
package Day2assignment;

public class Student {
	int stdID;
	String stdName;
	int stdClass;
	
	public void Display() {
		System.out.println(this.stdName);
		System.out.println(this.stdClass);
	}

	public static void main(String[] args) {
		
		Student s1 = new Student();
		s1.stdName = "Rishav";
		s1.stdClass = 12;
		s1.Display();
		
		Student s2 = new Student();
		s2.stdName = "Raj";
		s2.stdClass = 2;
		s2.Display();

	}

}

File: TypeScriptExample/Student.js
"use strict";
exports.__esModule = true;
exports.Student = void 0;
var Student = /** @class */ (function () {
    function Student() {
        this.stdid = 123;
        this.stdname = "Rishav";
        this.stdstandard = "12th Std";
    }
    Student.prototype.display = function () {
        console.log(this.stdid);
        console.log(this.stdname);
        console.log(this.stdstandard);
    };
    return Student;
}());
exports.Student = Student;
